tests-e2e/cypress/integration/adminconsole/analytics_spec.js

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// ***************************************************************
+
+describe('admin console > site statistics', () => {
+    let testUser;
+    let testTeam;
+    let testPlaybook;
+    let testSysadmin;
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user}) => {
+            testTeam = team;
+            testUser = user;
+            cy.apiCreateCustomAdmin().then(({sysadmin}) => {
+                testSysadmin = sysadmin;
+            });
+            cy.apiCreatePlaybook({
+                teamId: testTeam.id,
+                title: 'Playbook',
+                memberIDs: [],
+            }).then((playbook) => {
+                testPlaybook = playbook;
+            });
+        });
+    });
+
+    describe('site statistics', () => {
+        it('playbooks and runs counters are visible', () => {
+            cy.apiLogin(testSysadmin);
+
+            // # Go to admin console > site statistics
+            cy.visit('/admin_console/reporting/system_analytics');
+            cy.findByTestId('playbooks.playbook_count').should('exist');
+            cy.findByTestId('playbooks.playbook_run_count').should('exist');
+        });
+
+        it('playbook counter increases after creating a playbook', () => {
+            cy.apiLogin(testSysadmin);
+            let counter;
+
+            // # Go to admin console > site statistics
+            cy.visit('/admin_console/reporting/system_analytics');
+
+            // # Capture current value of playbook counter
+            cy.findByTestId('playbooks.playbook_count').invoke('prop', 'innerText').then((pbCount) => {
+                counter = parseInt(pbCount, 10);
+                cy.apiLogin(testUser);
+
+                // # Create a playbook
+                cy.apiCreatePlaybook({
+                    teamId: testTeam.id,
+                    title: 'Playbook',
+                    memberIDs: [],
+                }).then(() => {
+                    cy.apiLogin(testSysadmin);
+
+                    // # Go to admin console > site statistics
+                    cy.visit('/admin_console/reporting/system_analytics');
+
+                    // * Playbook Counter has been increased by 1
```

Comment:
Super nit: we usually start these sentences with a verb. It's not a written rule, just a convention we seem to follow naturally.
```suggestion
                    // * Verify that the Playbook Counter has been increased by 1
```